src/Col.js

```diff
@@ -78,6 +78,10 @@ const Col = (props) => {
     }
   });
 
+  if (attributes['xs']) {
```

Comment:
It was previously before the check, (see commit](https://github.com/reactstrap/reactstrap/pull/267/files#diff-771c8561c9896cbffb7d4b7f17506333)

![image](https://cloud.githubusercontent.com/assets/399776/21831442/b5e2651c-d758-11e6-93f4-17c008ad6233.png)